source/representation.lisp

```diff
@@ -55,6 +72,7 @@
 (defmethod princ-ccldoc ((c clause-object) stream)
   (let ((text (clause-text c)))
     (when (> (length text) 50)
+      ;; TODO Not setf??
```

Comment:
You can replace SETQ with SETF for slightly more modern code, but it is not illegal to use SETQ here.